library/src/scripts/theming/ThemeProvider.tsx

```diff
@@ -62,9 +66,29 @@ class BaseThemeProvider extends React.Component<IProps> {
     }
 }
 
+export function getLogo(logoType: LogoType): string | undefined {
+    const store = getStore<ICoreStoreState>();
+    const assets = store.getState().theme.assets.data || {};
+    let logo;
+
+    if (logoType === LogoType.DESKTOP) {
+        logo = assets.logo || null;
+    } else if (logoType === LogoType.MOBILE) {
+        logo = assets.mobileLogo || null;
+    }
+
+    if (!logo) {
+        return undefined;
```

Comment:
For consistencies sake can we return null out of this? I'm not the biggest fan of mixing null/undefined together when it's not necessary.